lib/src/HalstackContext.tsx

```diff
@@ -343,23 +337,24 @@ const parseTheme = (theme) => {
   wizardTokens.visitedHelperTextFontColor = theme?.wizard?.fontColor ?? wizardTokens.visitedHelperTextFontColor;
   wizardTokens.selectedHelperTextFontColor = theme?.wizard?.fontColor ?? wizardTokens.selectedHelperTextFontColor;
   wizardTokens.unvisitedStepBorderColor =
-    addLightness(theme?.wizard?.fontColor, 40) ?? wizardTokens.unvisitedStepBorderColor;
+    addLightness(40, theme?.wizard?.fontColor) ?? wizardTokens.unvisitedStepBorderColor;
   wizardTokens.unvisitedStepFontColor =
-    addLightness(theme?.wizard?.fontColor, 40) ?? wizardTokens.unvisitedStepFontColor;
+    addLightness(40, theme?.wizard?.fontColor) ?? wizardTokens.unvisitedStepFontColor;
   wizardTokens.unvisitedLabelFontColor =
-    addLightness(theme?.wizard?.fontColor, 40) ?? wizardTokens.unvisitedLabelFontColor;
+    addLightness(40, theme?.wizard?.fontColor) ?? wizardTokens.unvisitedLabelFontColor;
   wizardTokens.unvisitedHelperTextFontColor =
-    addLightness(theme?.wizard?.fontColor, 40) ?? wizardTokens.unvisitedHelperTextFontColor;
+    addLightness(40, theme?.wizard?.fontColor) ?? wizardTokens.unvisitedHelperTextFontColor;
 
   return componentTokensCopy;
 };
 
-const parseLabels = (labels) => {
-  const parsedLabels = defaultTranslatedComponentLabels;
+const parseLabels = (labels: DeepPartial<TranslatedLabels>): TranslatedLabels => {
+  const parsedLabels = JSON.parse(JSON.stringify(defaultTranslatedComponentLabels));
+
   Object.keys(labels).map((component) => {
     if (parsedLabels[component]) {
       Object.keys(parsedLabels[component]).map((label) => {
-        if (labels[component][label]) {
+        if (labels[component][label] && labels[component]![label]) {
           parsedLabels[component][label] = labels[component][label];
```

Comment:
Why is this condition added to the if statement? `labels[component]![label]`

Reply:
It's the typescript operator `!`. Checks if the value before it is null or not, which in this case doesn't make any sense since it is already checked in the previous condition. I don't know why I add that, I'm removing it.